.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Main CI workflow
+# Note: If the workflow name is changed, the CI badge URL in the README must also be updated
+
+on:
+  push:       # Push trigger runs on any pushed branch.
+  schedule:   # Scheduled trigger runs on the latest commit on the default branch.
+    - cron:  '0 22 * * *'
```

Comment:
The trigger time has changed, but I assume that doesn't matter?

Reply:
Yeah it used to be "5 minutes past the hour", but it seemed pointless so I just simplified it to "on the hour".